winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -25,11 +25,33 @@ private WinRmTool(String address, String username, String password) {
         this.password = password;
     }
 
+    /**
+     * Execute a list of Windows Native commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"&"</code> delimiter and a terminating one.
+     * @deprecated since 0.2; Use the {@link #executeCommand(String)} instead and transform your commands list explicitly
+     */
+    @Deprecated
     public WinRmToolResponse executeScript(List<String> commands) {
-        return executeScript(joinScript(commands));
+        return executeCommand(joinCommands(commands));
```

Comment:
Good idea using `&` as the delimiter. Since it's the recommended approach to chaining commands we can now provide a well behaved `executeCommand(List)`, wdyt?